frb/galaxies/eazy.py

```diff
@@ -166,6 +168,10 @@ def eazy_input_files(photom, input_dir, name, out_dir, prior_filter=None,
     if cosmo is None:
         cosmo = Planck15
 
+    # Convert to table if input photom is a dict.
+    if type(photom)==dict:
```

Comment:
I'd start using `isinstance`

I think it is preferred, e.g.

`if isinstance(photom, dict):`

Reply:
although in this case, I think you can just do:

`photom = Table(photom)` as a Table will recast to itself..

Reply:
Changed.